.github/workflows/build-test.yml

```diff
@@ -67,22 +67,22 @@ jobs:
                   CLOUD_PROVIDER: ${{ matrix.cloud_provider }}
                   PARAMETERS_SECRET: ${{ secrets.PARAMETERS_SECRET }}
               run: ci\test_win.bat
-#    build-test-mac:
-#        name: Build-Test-Mac
-#        runs-on: macos-10.15
-#        strategy:
-#            matrix:
-#                build_type: [ 'Debug' ]
-#                cloud_provider: [ 'AWS' ]
-#        steps:
-#            - uses: actions/checkout@v1
-#            - name: Install Homebrew Bash
-#              shell: bash
-#              run: brew install bash
-#            - name: Build
-#              shell: bash
-#              env:
-#                PARAMETERS_SECRET: ${{ secrets.PARAMETERS_SECRET }}
-#                BUILD_TYPE: ${{ matrix.build_type }}
-#                CLOUD_PROVIDER: ${{ matrix.cloud }}
-#              run: /usr/local/bin/bash ./ci/build_mac.sh
+   build-test-mac:
+       name: Build-Test-Mac
+       runs-on: macos-10.15
```

Comment:
Are we going to support 10.15?

Reply:
I changed it to macos-latest but build fails on curl build